### DISCUSSION TITLE
Just show feature subset in the input editor in What-if tab?

### DISCUSSION THREAD

**Discussion by tonydengpace** [oegedijk/explainerdashboard#170] [Q&A]
Hi. For the feature input in the What-if tab, would it be possible to just show a subset of the features instead of the full feature set?

**Comment by oegedijk** [comment]
Euhm, this could be added as a keyword argument perhaps. So you could add a list of features to use in the whatif tab with e.g. `ExplainerDashboard(explainer, what_if_features = ['Age', 'PassengerClass']).run()`.

Shouldn't be too hard to implement, but may require a bit of work. The input component is defined here: https://github.com/oegedijk/explainerdashboard/blob/bdd03da0edd6efb48876e539bc0669101c6bb990/explainerdashboard/dashboard_components/overview_components.py#L633

I think you would have to keep a seperate list of visible features and hidden features, and then slice only through the visible features for the display. May not be too hard.

Unfortunately I have extremely limited time to work on this right now. You want to have a stab at it?
